lib/vehicle.rb

```diff
@@ -0,0 +1,32 @@
+class Vehicle
+  attr_reader :year, :make, :model, :vehicle, :passengers, :num_adults
+  def initialize(year, make, model, speeding = false, passengers = [])
+    @year = year
+    @make = make
+    @model = model
+    @speeding = speeding
+    @passengers = passengers
+  end
```

Comment:
```suggestion
  def initialize(year, make, model)
    @year = year
    @make = make
    @model = model
    @speeding = false
    @passengers =[]
  end
```

Your way works but unless we expect we'll sometimes manually be passing in different values for `speeding` or `passengers` it's more conventional to just set their values in line.